jiant/trainer.py

```diff
@@ -1034,8 +1046,7 @@ def _check_stop(self, val_n, stop_metric, tasks):
         return should_stop
 
     def _forward(self, batch, task=None):
-        tensor_batch = move_to_device(batch, self._cuda_device)
-        model_out = self._model.forward(task, tensor_batch)
+        model_out = self._model.forward(task, batch)
```

Comment:
not related to this PR, but we should probably delete this at some point, it's a hold-over from when I was importing this from AllenNLP but we don't seem to have a good reason to keep this extra step around

Reply:
agreed